app/views/users/show.html.erb

```diff
@@ -1,39 +1,36 @@
-<main>
-  <section>
-    <div class="card">
-      <div class="user-image border">
-        <%= image_tag(@user.photo, class: "add-css-class") %>
-      </div>
-      <div class="user-data border">
-        <h1><%= @user.name %></h1>
-        <p class="number-posts">Number of posts: <%= @user.post_counter %></p>
-        <div class="right">
-          <%= link_to "Add a post", new_user_post_path(@user), class: "btn right" %>
-        </div>
-      </div>
+<div class="bg-dark min-vh-100 text-white" >
+  <div class="container px-2 py-5">
+  
+  <%= render 'user' %>
+
+  <h1><%=@current_user.name%>  </h1>
+  
+  <div class="bg-secondary px-4 py-3 mb-3" >
+    <h2>Bio</h2>
+    <p> <%=@user.bio%> </p>
+  </div>
+
+  
+  <% @user.recents_posts.each do |post| %>
```

Comment:
- [ ] This should be `recent_posts`.

  ![image](https://github.com/Marcraphael12/rails-blog-app/assets/36542986/3db4cd48-3307-42f8-88e8-1e67bb690b82)